common/common-rest/src/main/java/org/apache/servicecomb/common/rest/AbstractRestInvocation.java

```diff
@@ -249,4 +265,21 @@ protected void onExecuteHttpServerFiltersFinish(Response response, Throwable e)
       invocation.onFinish(response);
     }
   }
+
+  public void setAfterCreateInvocationHandler(
+      AfterCreateInvocationHandler afterCreateInvocationHandler) {
+    this.afterCreateInvocationHandler = afterCreateInvocationHandler;
+  }
+
+  /**
+   * After the {@link AbstractRestInvocation#invocation} is instantiated, the {@link AbstractRestInvocation#invocation}
+   * will be passed into {@link AfterCreateInvocationHandler#handle(Invocation)} so that users can do something with it.
+   */
+  public static interface AfterCreateInvocationHandler {
+    /**
+     * @see AfterCreateInvocationHandler
+     * @param invocation {@link AbstractRestInvocation#invocation}
```

Comment:
This document refers itself, and can be deleted

Reply:
`AfterCreateInvocationHandler` has been removed.